app/models/Reward.java

```diff
@@ -10,14 +11,15 @@
 public class Reward {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    public Long id;
-    public int rewardId;
-    public RewardType rewardType;
-    public ArrayList<Integer> points;
-    public ArrayList<Integer> money;
+    private Long id;
+    private int rewardId;
+    private RewardType rewardType;
+    private ArrayList<Integer> points;
+    private ArrayList<Integer> money;
 
     @OneToMany(mappedBy="judgment", cascade= CascadeType.ALL)
-    public List<Judgment> judgmentRiderConnections = new ArrayList<Judgment>();
+    @JsonManagedReference
+    private List<Judgment> judgmentRiderConnections = new ArrayList<Judgment>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)